installer/templates/new/web/views/error_view.ex

```diff
@@ -13,5 +13,17 @@ defmodule <%= application_module %>.ErrorView do
   # template is found, let's render it as 500
   def template_not_found(_template, assigns) do
     render "500.html", assigns
+  end<% else %>def render("404.json", _assigns) do
+    %{"errors" => %{"detail" => "Page not found"}}
   end
+
+  def render("500.json", _assigns) do
+    %{"errors" => %{"detail" => "Server internal error"}}
```

Comment:
should we prefer string keys here? My personal preference is atom keys when constructing json maps by hand. Strings everywhere is more consistent, but a bit cumbersome.